spec/bolt/logger_spec.rb

```diff
@@ -45,7 +45,7 @@ def initialize(*args)
     it 'submits an analytics event' do
       expect(analytics).to receive(:event).with('Warn', 'deprecation', { label: "We've got clearance Clarence" })
       Bolt::Logger.analytics = analytics
-      Bolt::Logger.deprecation_warning("We've got clearance Clarence", 'Roger Roger')
```

Comment:
This should also include a test for the logic in `self.issue_queued_messages`, particularly that the message queue gets cleared at the end.